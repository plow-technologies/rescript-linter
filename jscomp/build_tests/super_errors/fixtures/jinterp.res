
let a = 11
let b = j`number $(a)`