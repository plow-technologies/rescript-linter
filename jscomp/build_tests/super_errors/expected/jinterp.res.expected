
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/jinterp.res[0m:[2m3:10-21[0m

  1 [2mâ”‚[0m 
  2 [2mâ”‚[0m let a = 11
  [1;31m3[0m [2mâ”‚[0m let b = j[1;31m`number $(a)[0m`

  The unsafe j`$(a)$(b)` interpolation was removed, use string template `${a}${b}` instead.